.github/workflows/main.yml

```diff
@@ -688,10 +688,19 @@ jobs:
           check_name: VSCode Test Report
         if: (steps.test_vscode.outcome == 'failure' || steps.test_vscode.outcome == 'failure') && failure()
 
+      - name: Set xvfb command linux
+        if: matrix-os == 'ubuntu-latest'
+        run: echo "xvfbCommand=--server-args=""-screen 0 1024x768x24"""" >> $GITHUB_ENV
+
+      - name: Set xvfb command non-linux
```

Comment:
Could you just skip this entire step?

Reply:
${{ env.xvfbCommand }} when env.xfvbCommand is not defined is the same as the empty string I think